auth-api/src/auth_api/services/api_gateway.py

```diff
@@ -140,29 +142,32 @@ def get_api_keys(cls, org_id: int) -> List[Dict[str, any]]:
         check_auth(one_of_roles=(ADMIN, STAFF), org_id=org_id)
         api_keys_response = {'consumer': {'consumerKey': []}}
         for email in cls._get_email_id(org_id, 'sandbox'), cls._get_email_id(org_id, 'prod'):
-
             try:
                 consumers_response = RestService.get(
                     f'{consumer_endpoint}/mc/v1/consumers/{email}',
                     additional_headers={'x-apikey': gw_api_key}
                 )
                 keys = consumers_response.json()['consumer']['consumerKey']
-                cls._filter_and_add_keys(api_keys_response, keys)
+                cls._filter_and_add_keys(api_keys_response, keys, email)
             except HTTPError as exc:
                 if exc.response.status_code != 404:  # If consumer doesn't exist
                     raise exc
 
         return api_keys_response
 
     @classmethod
-    def _filter_and_add_keys(cls, api_keys_response, keys):
+    def _filter_and_add_keys(cls, api_keys_response, keys, email):
+        def _add_key_to_response(_key):
+            if _key['keyStatus'] == 'approved':
```

Comment:
I hope this is the correct keystatus to filter out revoked keys

Reply:
Yes, revoked keys comes a revoked keyStats